.travis.yml

```diff
@@ -1,7 +1,34 @@
 language: python
 python:
   - "2.7"
+git: 
+    depth: 1
+
+addons:
+  postgresql: '9.4'
+
+install:
+    - yes | python setup.py install
+    - pip install pytest
+    - pip install pyasn1
 before_script:
+    - echo -e "machine github.com\n  login $CI_USER_TOKEN" >> ~/.netrc
+    - cat ~/.netrc
+    - cd ..
+    - git clone https://github.com/uc-cdis/gdcapi.git
+
+    - cd gdcapi
+    - git checkout feat/dd-tests
+    - ./setup.sh
     - yes | python setup.py install
-    - pip freeze
-script: "nosetests -v"
+    - pip install -r requirements.txt
+    - pip install -r dev-requirements.txt
+    - pip install -e "git+https://$CI_USER_TOKEN@github.com/uc-cdis/user-api.git@78f5bbdeb9507318af5d4f81479ecf1150b5e266#egg=userapi"
```

Comment:
If you run this before `pip install -r dev-requirements.txt` it should work without removing that from gdcapi?